### PR TITLE
I0419 Create Bridge.nuget's uninstall.ps1 script 

### LINKED ISSUE #419: Lots of red squiggles in VS using Bridge (System.Void not defined, dupe references between Bridge+mscorlib)
Many of my colleagues have reported this to me for a while, but I hadn't really seen it. Today, I'm seeing it, and I can't find any way to fix it.

We have a bunch of Bridge code that works, but VS reports lots of errors (mostly `Predefined type 'System.Object/Void' is not defined or imported`, but also some types defined in multiple places). There are no references to any MS `System` assemblies (screenshot includes reference below).

The repo I'm building is available here:
https://bitbucket.org/DanRoberts/reactbridgedotnet

Squiggles

Note: The project "works", but finding real compile errors will be tricky like this!

**UPDATE** by @DaniilVeriga:
Confirmed to be a [bug](https://connect.microsoft.com/VisualStudio/feedback/details/1893825/targets-file-from-a-nuget-package-is-not-re-applied-to-the-project-after-nuget-restore) in Visual Studio by Microsoft team. They are looking to fix in the next major version of VS (after 2015).

### PR DESCRIPTION
**Pull request by fabriciomurta**
Fixes #419 .

Changes proposed in this pull request:

- Adds code to install.ps1 to set project's NoStdLib flag to true during NuGet install process.
- Adds uninstall.ps1 script to revert NoStdLib settings to false on uninstallation of the NuGet package.
- Changes the open project without the need to reload it -- nor force saving it until the user saves the project from his own.

Assigning to @geoffreymcgill. If he's ok with the change, then proceed with the pull request (probably by assigning to @LeonidVeriga).

### CODE REVIEW

**Comment by geoffreymcgill** [comment]
Also related to #1057.

**Comment by LeonidVeriga** [comment]
@fabriciomurta 

1) Create a new class library project in VS2015,  `Save` the Project.
Active build configuration is default (`Debug`/`Any CPU`).
Content of the csproj file (unnecessary details skipped):
```
 <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
 </PropertyGroup>
 <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
 </PropertyGroup>
```

2) `install-package Bridge`, `Save` the Project
```
 <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
    <NoStdLib>true</NoStdLib>
 </PropertyGroup>
 <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
 </PropertyGroup>
```
`NoStdLib` is for `Debug` only. But rebuilding solution works for both `Debug` and `Release` configurations as `StdLib` is also in `bridge.targets`

3) `uninstall-package Bridge`, `Save` the Project
```
 <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
    <NoStdLib>false</NoStdLib>
 </PropertyGroup>
 <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
    <NoStdLib>false</NoStdLib>
 </PropertyGroup>
```

4) `install-package Bridge`, `Save` the Project
```
 <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
    <NoStdLib>true</NoStdLib>
 </PropertyGroup>
 <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
    <NoStdLib>true</NoStdLib>
 </PropertyGroup>
```
`Package Manager Console` output contains error:
```
Removing Reference to System.Core
Exception calling "Remove" with "0" argument(s): "This reference cannot be removed from the project because it is always referenced by the compiler."
At ...\install.ps1:10 char:9
+         $_.Remove()
+         ~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ComMethodTargetInvocation
```

But the installation at whole is successful.

File: .build/files/uninstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿#First some common params, delivered by the nuget package installer
+param($installPath, $toolsPath, $package, $project)
+
+Write-Host ("Removing Bridge.NET specific settings on '" + $project.ProjectName)
+
+# Sets the NoStdLib setting back to default for every project configuration.
+# See Issue #419 for more information on why.
+# https://github.com/bridgedotnet/Bridge/issues/419
+# Once Visual Studio or NuGet defect is fixed, lines 10-216 can be removed.
```
Comment (DanTup) [inline .build/files/uninstall.ps1]: Typo: "lines 10-216" should say "lines 10-16"?

(6-16 is probably better; though you could arguably remove the entire file the way it currently stands!) :)

Comment (LeonidVeriga) [inline .build/files/uninstall.ps1]: Thanks @DanTup, done)

**Pull request merged**